packages/xfce4/build.sh

```diff
@@ -1,9 +1,9 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/termux/x11-packages
+TERMUX_PKG_HOMEPAGE=https://xfce.org/
 TERMUX_PKG_DESCRIPTION="A metapackage that installs all parts of the XFCE desktop environment"
 TERMUX_PKG_LICENSE="Public Domain"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.1
-TERMUX_PKG_REVISION=16
+TERMUX_PKG_VERSION=4.16
+TERMUX_PKG_REVISION=17
```

Comment:
The TERMUX_PKG_REVISION line can be removed since TERMUX_PKG_VERSION is increased

Reply:
ok, I'll keep making more changes and remove the draft when I'm done.